packages/ia-topnav/src/media-button.js

```diff
@@ -103,6 +103,7 @@ class MediaButton extends TrackedElement {
         @click=${this.followable ? this.trackClick : this.onClick}
         data-event-click-tracking="${this.analyticsEvent}"
         title="${this.tooltipPrefix} ${this.mediatype} menu"
+        tabindex="${this.openMenu === 'media' ? '' : '0'}"
```

Comment:
Q: Does this approach add and remove `tabindex` from elements dynamically?

Reply:
Its seems not usable change. reverting this. Thanks!